tokenizers/src/tokenizer/mod.rs

```diff
@@ -876,3 +900,35 @@ impl Tokenizer {
         }
     }
 }
+
+#[derive(Debug)]
+pub struct Lines<B> {
```

Comment:
We can probably move this somewhere under `utils`. It might be interesting to be able to use it somewhere else.

Reply:
For sure !